backend/docs/api/inventory.openapi.yaml

```
@@ -0,0 +1,1752 @@
+openapi: 3.0.1
+info:
+  title: Device inventory
+  description: >
+    An API for uploading device attributes.
+
+
+    Devices can upload vendor-specific attributes (software/hardware info,
+    health checks, metrics, etc.) of various data types to the backend.
+  version: '1'
+
+tags:
+  - name: Inventory Device
+  - name: Inventory Internal
```

```suggestion
  - name: Inventory Device v1
  - name: Inventory Internal v1
```
total nit and feel free to ignore, just thought about the fact that it is v1

---

also it is "Inventory Device" here and `Inventory Devices`  in the path def below